packages/components/src/components/assets/Icon/Icon.tsx

```diff
@@ -117,22 +117,14 @@ export const Icon = React.forwardRef(
         const theme = useTheme();
         const defaultColor = color ?? theme.TYPE_LIGHT_GREY;
 
-        const handleOnKeyDown = (e: KeyboardEvent) => {
-            if (e.key === 'Enter' || e.key === ' ') {
-                onClick?.(e as any);
-            }
-        };
-
```

Comment:
This is also used in `CollapsibleBox`.

Reply:
It uses the shared `Icon` from `@suite-common/icons/src/webComponents` ;)